cluster/examples/kubernetes/ceph/monitoring-roles.yaml

```diff
@@ -0,0 +1,79 @@
+#
+# OLM: BEGIN ROLE
+---
+# Aspects for creation of monitoring resources
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-monitor
+  namespace: rook-ceph
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+# OLM: END ROLE
```

Comment:
Have you tested these files? I often see errors if there is a blank yaml definition like this (two triple-dashes with no content in between).